jf_agent/main.py

```diff
@@ -200,9 +200,6 @@ def main():
             diagnostics.capture_run_args(
                 args.mode, args.config_file, config.outdir, args.prev_output_dir
             )
-            ingest_config = get_ingest_config(
-                config=config, creds=creds, endpoint_jira_info=jellyfish_endpoint_info.jira_info
-            )
```

Comment:
Move ingest config out of here, where we generate it every time. Move it to the logic block that only triggers if the config flag is set